src/collection/views/EditCollection.tsx

```diff
@@ -343,7 +343,7 @@ const EditCollection: FunctionComponent<EditCollectionProps> = props => {
 
 		return currentCollection ? (
 			<Fragment>
-				<Container background="alt" mode="vertical" size="small">
+				<Container background={'alt'} mode="vertical" size="small">
```

Comment:
`background="alt"` is toch correct?

Reply:
ja, mijn editor probeert gewoon slim te zijn en als ik het zo `{'alt'}` zet, dan stopt hij met zagen
![image](https://user-images.githubusercontent.com/1710840/65768314-f49b2980-e130-11e9-9070-5d9e56070d37.png)

sorry ocd :p